src/Symfony/Bridge/Doctrine/DependencyInjection/AbstractDoctrineExtension.php

```diff
@@ -85,6 +87,7 @@ protected function loadMappingInformation(array $objectManager, ContainerBuilder
                     throw new \InvalidArgumentException(sprintf('Bundle "%s" does not exist or it is not enabled.', $mappingName));
                 }
 
+                // $mappingConfig = $this->getBundleMappingDriverDefaultConfig($mappingConfig, $bundleMetadata, $container);
                 $mappingConfig = $this->getMappingDriverBundleConfigDefaults($mappingConfig, $bundle, $container);
```

Comment:
Changing this call is where I'm stumbling on the compat layer.  In 2 of the 4 Doctrine extensions, they are extending this method before calling the parent, so flat out not calling the deprecated method could be problematic here.